store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/retriever/impl/AccumuloAdjacentIdRetriever.java

```diff
@@ -52,21 +54,22 @@
 import java.util.NoSuchElementException;
 import java.util.Set;
 
-public class AccumuloAdjacentIdRetriever extends AccumuloRetriever<GetAdjacentIds, EntityId> {
+public class AccumuloAdjacentIdRetriever extends AccumuloRetriever<EntityId> {
     private static final Logger LOGGER = LoggerFactory.getLogger(AccumuloAdjacentIdRetriever.class);
 
     private final Iterable<? extends ElementId> ids;
     private final Set<String> transformGroups;
 
-    public AccumuloAdjacentIdRetriever(final AccumuloStore store, final GetAdjacentIds operation,
+    public AccumuloAdjacentIdRetriever(final AccumuloStore store, final Operation operation, final View view, final DirectedType directedType,
                                        final User user)
             throws IteratorSettingException, StoreException {
-        super(store, operation, user,
-                store.getKeyPackage().getIteratorFactory().getEdgeEntityDirectionFilterIteratorSetting(operation),
-                store.getKeyPackage().getIteratorFactory().getElementPreAggregationFilterIteratorSetting(operation.getView(), store),
-                store.getKeyPackage().getIteratorFactory().getQueryTimeAggregatorIteratorSetting(operation.getView(), store),
-                store.getKeyPackage().getIteratorFactory().getElementPostAggregationFilterIteratorSetting(operation.getView(), store));
-        this.ids = operation.input();
+
+        super(store, operation, view, user,
+                store.getKeyPackage().getIteratorFactory().getEdgeEntityDirectionFilterIteratorSetting(operation, view, directedType),
```

Comment:
`Operation` parameter here might lead to needing to duplicate cast outside of the Handlers zone.

Reply:
Yeah, I was wondering what your thoughts were on this? You said to remove the input type from the class definition but this leads casting

Reply:
At the moment in side this method you casting to `SeededGraphFilters` leave it while it exists. When that class is removed then the next guy can sort it out. which might be you, but Im aiming for breadth-first changes rather than deep changes.